python_modules/libraries/dagster-snowflake/dagster_snowflake/db_io_manager.py

```diff
@@ -94,7 +94,6 @@ def handle_output(self, context: OutputContext, obj: object) -> None:
             )
```

Comment:
this is a random change - probably an accidental line deletion on my part. I'll remove it